src/main/java/uk/ac/ebi/eva/pipeline/parameters/validation/step/VariantLoaderStepParametersValidator.java

```diff
@@ -0,0 +1,78 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.pipeline.parameters.validation.step;
+
+import org.springframework.batch.core.JobParameters;
+import org.springframework.batch.core.JobParametersInvalidException;
+import org.springframework.batch.core.JobParametersValidator;
+import org.springframework.batch.core.job.CompositeJobParametersValidator;
+import org.springframework.batch.core.job.DefaultJobParametersValidator;
+
+import uk.ac.ebi.eva.pipeline.parameters.JobParametersNames;
+import uk.ac.ebi.eva.pipeline.parameters.validation.ConfigChunkSizeValidator;
+import uk.ac.ebi.eva.pipeline.parameters.validation.DbCollectionsFilesNameValidator;
+import uk.ac.ebi.eva.pipeline.parameters.validation.DbCollectionsVariantsNameValidator;
+import uk.ac.ebi.eva.pipeline.parameters.validation.DbNameValidator;
+import uk.ac.ebi.eva.pipeline.parameters.validation.InputStudyIdValidator;
+import uk.ac.ebi.eva.pipeline.parameters.validation.InputVcfAggregationValidator;
+import uk.ac.ebi.eva.pipeline.parameters.validation.InputVcfIdValidator;
+import uk.ac.ebi.eva.pipeline.parameters.validation.InputVcfValidator;
+import uk.ac.ebi.eva.pipeline.parameters.validation.OptionalValidator;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Validates the job parameters necessary to execute a
+ * {@link uk.ac.ebi.eva.pipeline.jobs.steps.PopulationStatisticsGeneratorStep}
+ */
+public class VariantLoaderStepParametersValidator extends DefaultJobParametersValidator {
+
+    public VariantLoaderStepParametersValidator() {
+        super(new String[]{
+                JobParametersNames.DB_NAME,
+                JobParametersNames.DB_COLLECTIONS_VARIANTS_NAME,
+                JobParametersNames.DB_COLLECTIONS_FILES_NAME,
+                JobParametersNames.INPUT_STUDY_ID,
+                JobParametersNames.INPUT_VCF_ID,
+                JobParametersNames.INPUT_VCF,
+        }, new String[]{});
+
+    }
+
+    @Override
+    public void validate(JobParameters parameters) throws JobParametersInvalidException {
+        super.validate(parameters);
+        compositeJobParametersValidator().validate(parameters);
+    }
+
+    private CompositeJobParametersValidator compositeJobParametersValidator() {
+        final List<JobParametersValidator> jobParametersValidators = Arrays.asList(
+                new DbNameValidator(),
+                new DbCollectionsVariantsNameValidator(),
+                new DbCollectionsFilesNameValidator(),
+                new InputStudyIdValidator(),
+                new InputVcfIdValidator(),
+                new InputVcfValidator(),
+                new OptionalValidator(new InputVcfAggregationValidator(), JobParametersNames.INPUT_VCF_AGGREGATION),
```

Comment:
At the moment it is necessary for both. the VariantLoaderStep (which GenotypedJob and AggregatedJob share) currently creates a bean for the VcfReader (AggregatedVcfReader or  VcfReader depending on this parameter). In some previous version, it was the job who defined the reader, but with @jorizci we decided this design was a bit better.

I think that making the aggregation not mandatory is not trivial, and may involve one of:
- default NONE in JobOptions, required for any aggregated, even BASIC
- not required for NONE or BASIC, but the jobs add this parameter to the listener (which modifies the JobOptions so the step instantiates the correct one).
- stop sharing the VariantLoaderStep in both jobs (this would make me very sad)